connector/src/main/java/org/jboss/as/connector/subsystems/datasources/AbstractDataSourceAdd.java

```diff
@@ -206,6 +207,8 @@ void firstRuntimeStep(OperationContext context, ModelNode operation, ModelNode m
                 dataSourceService.getDriverInjector());
 
          // If the authentication context is defined, add the capability
+         // TODO ELYTRON_ENABLED is redundant as AUTHENTICATION_CONTEXT implies elytron
```

Comment:
It's not redundant because if you don't set AUTHENTICATIONCONTEXT and ELYTRON_ENABLED == true the current AUTHENTICATION_CONTEXT (for example from ejb) will be used